string[] axes
float64[] deltas
float64[] durations
---
bool ok
string message
